src/games/arena/repositories/arena/actions/admin/create-or-finish-game.ts

```diff
@@ -38,7 +38,7 @@ export async function newGame(commandText: string, userRequesting: User) {
       transaction
     );
 
-    await enableAllItems(GAME_TYPE.ARENA, transaction);
+    await enableAllItems(game._gameTypeId, transaction);
```

Comment:
changed all the functions where gameTypeId was string and used the new int id

Reply:
Amazing